requirements.txt

```diff
@@ -11,8 +11,8 @@
 sphinx
 cwrap
 numpy
-pandas
-matplotlib<3
+pandas<0.25 # Pandas > 0.25 is not possible on RHEL6 since it requires some libs to be provided by the os
```

Comment:
But _ERT_ targets other platforms then RHEL 6. Is this a restriction of _ERT_ or of RHEL 6? If it is the first then the bound should stay, if it is the later this is the responsibility of the deployer.

Reply:
This was implemented pre our discussion with regards to komodo reinstalling libs and its deps and by that bumped pandas through 0.25 and broke our environment. I will remove this commit, and make a new issue in komodo adressing this which should be solved with --no-deps and with us defining all versions of all deps.